### PR TITLE
DASH-969 Add buy fio tokens tile

### PR DESCRIPTION
**Pull request by sardonyxwt**

### CODE REVIEW

File: client/src/pages/WalletsPage/WalletsPage.tsx [hunk 1]
```diff
@@ -156,16 +157,11 @@ const WalletsPage: React.FC<Props> = () => {
             />
           </div>
         )}
-        <TotalBalanceBadge
```
Comment (trukhilio) [inline client/src/pages/WalletsPage/WalletsPage.tsx:159]: Why did you remove this. It should be used for mobile UI.

File: client/src/pages/WalletsPage/components/NeedTokensBadge.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import classnames from 'classnames';
+
+import FioTokens from '../../../assets/images/fio-tokens.png';
+
+import classes from '../styles/NeedTokensBadge.module.scss';
+import config from '../../../config';
+
+const TotalBalanceBadge: FC = () => {
```
Comment (trukhilio) [inline client/src/pages/WalletsPage/components/NeedTokensBadge.tsx:9]: Rename component to NeedTokensBadge. Use named export here.

**Review by trukhilio — CHANGES_REQUESTED** [review]
Check comments

File: client/src/pages/WalletsPage/WalletsPage.tsx (hunk 1 above)
Comment (psystrpla) [reply client/src/pages/WalletsPage/WalletsPage.tsx:159]: No in mobile view we have two blocks (Bug I think)

Comment (trukhilio) [reply client/src/pages/WalletsPage/WalletsPage.tsx:159]: It's bug to show 2-nd one. Main reason is that to show balance after the list of wallets before Welcome component.
Screenshot 2024-08-01 at 14 39 00

Comment (sardonyxwt) [reply client/src/pages/WalletsPage/WalletsPage.tsx:159]: We have duplicate block in mobile view

Comment (trukhilio) [reply client/src/pages/WalletsPage/WalletsPage.tsx:159]: For mobile view let's remove 2nd one.

**Review by trukhilio — APPROVED** [review]

**Pull request merged**